v4/source/commands.yaml

```diff
@@ -155,6 +155,46 @@
             listCommands, resp := Client.ListAutocompleteCommands(<TEAMID>)
 
   '/commands/{command_id}':
+    get:
+      tags:
+        - commands
+      summary: Get a command
+      description: |
+        Get a command definition based on command id string.
+        ##### Permissions
+        Must have `manage_slash_commands` permission for the team the command is in.
+
+        __Minimum server version__: 5.20
```

Comment:
why 5.20?

Reply:
Hmmm.  Good point.  I assumed the server endpoint would be going into next release but I'll have to confirm the release version.  Thanks.

Reply:
Bumped minimum version to 5.22